charts/kubearchive/templates/operator/kubearchiveconfig_editor_role.yaml

```
@@ -5,12 +5,11 @@ kind: ClusterRole
 metadata:
   labels:
     app.kubernetes.io/name: clusterrole
-    app.kubernetes.io/instance: kubearchiveconfig-editor-role
+    app.kubernetes.io/instance: {{ .Values.kubearchive.namespace }}config-editor-role
```

This looks like it is missing a dash.

```suggestion
    app.kubernetes.io/instance: {{ .Values.kubearchive.namespace }}-config-editor-role
```